### ISSUE TITLE
Dropping materialized views from migrations

### ISSUE THREAD

**Issue by Drenmi** [thoughtbot/scenic#222]
Firstly, thank you for your hard work on this amazing library. 🙂 

Now on to my predicament. It seems that `#drop_materialized_view` has not been made available from within migrations. Using `#drop_view` will also not work on a materialized view, since Postgres is particular about how you do this.

### Expected Behaviour

The following should drop the materialized view `aggregate_scores`:

```ruby
class DropAggregateScores < ActiveRecord::Migration[5.0]
  def change
    drop_materialized_view :aggregate_scores
  end
end
```

### Actual Behaviour

Using `#drop_materialized_view`:

```
NoMethodError: undefined method `drop_materialized_view' for #<DropAggregateScores:0x007fa464bd8110>
```

Using `#drop_view`:

```
ActiveRecord::StatementInvalid: PG::WrongObjectType: ERROR:  "aggregate_scores" is not a view
HINT:  Use DROP MATERIALIZED VIEW to remove a materialized view.
```

### Steps to Reproduce

1. Create a materialized view.
2. Try dropping it from a migration using `#drop_materialized_view` or `#drop_view`.

### Version

scenic 1.4.0

**Comment by devonestes** [comment]
Hey!

I think what you're trying to do can be accomplished with the following:


```ruby
class DropAggregateScores < ActiveRecord::Migration[5.0]
  def change
    drop_view :aggregate_scores, materialized: true
  end
end
```

Want to give that a whirl?

**Comment by Drenmi** [comment]
@devonestes: That does indeed work! Perhaps the `README` should be updated to include this? 🙂

**Comment by devonestes** [comment]
@Drenmi Do you think #223 would be sufficient?

**Comment by calebthompson** [comment]
Resolved by #223, but let me know if you disagree @Drenmi

**Comment by Drenmi** [comment]
Looks great! No objections. Thanks for the effort. 🙂
